src/pages/_error.jsx

```diff
@@ -61,7 +56,7 @@ function Error({ statusCode, proxiedProductSlug, layoutProps }) {
 	const Layout = isProxiedDotIo ? ProxiedLayout : BaseLayout
 
 	return (
-		<Layout data={{ ...layoutProps }}>
```

Comment:
As with `_app.tsx`, the `io`-specific Dato-specific `layoutProps` are no longer needed here, as the only `io`-ish site we render is Sentinel, and it does not require data fetched from Dato.